src/popup/router/pages/Send.vue

```diff
@@ -229,15 +249,29 @@ export default {
       }
       this.loading = true;
       try {
-        const { hash } = await this.sdk.spend(amount, receiver, { waitMined: false, modal: false });
-        if (hash) {
+        if (this.selectedToken) {
+          const { hash } = await this.$store.dispatch('fungibleTokens/transfer', [
+            receiver,
+            this.form.amount,
+          ]);
+          this.$store.commit('addPendingTransaction', {
+            hash,
+            amount,
+            type: 'spendToken',
+            recipientId: receiver,
+          });
+        } else {
+          const { hash } = await this.sdk.spend(amount, receiver, {
+            waitMined: false,
+            modal: false,
```

Comment:
```suggestion
```
the same as you did here
https://github.com/aeternity/superhero-wallet/pull/682/files#diff-1a443786f57caf4444a2d47ba9eb8dcbdee7c2d2bfe4165d2889f65825771fe4R123

Reply:
Actually this parameter shows/hides the confirmation modal. I will get it back it is breaking the integration test.